src/main.cpp

```diff
@@ -862,16 +870,50 @@ bool CheckTransaction(const CTransaction& tx, CValidationState &state,
     }
 }
 
-bool CheckTransactionWithoutProofVerification(const CTransaction& tx, CValidationState &state)
+bool CheckTransactionWithoutProofVerification(const CTransaction& tx, CValidationState &state, uint32_t consensusBranchId)
 {
     // Basic checks that don't depend on any context
 
-    // Check transaction version
-    if (tx.nVersion < MIN_TX_VERSION) {
+    // Check min and max tx version of pre-Overwinter and Overwinter transactions.  Range allows for soft fork.
```

Comment:
Defining a range allows for soft-forked transaction versions, but the current values of the Overwinter range endpoints do not. Is there a clearer way to phrase this?